src/test/java/org/openlmis/buq/service/buq/BottomUpQuantificationServiceTest.java

```diff
@@ -393,6 +399,53 @@ public void shouldThrowValidationMessageExceptionIfUserHomeFacilityIsNotEqualsTo
     bottomUpQuantificationService.delete(bottomUpQuantification);
   }
 
+  @Test
+  public void shouldReturnApproveFacilityForecastingStatsWithCalculatedValues() {
+    UserDto user = new UserDtoDataBuilder().buildAsDto();
+    when(authenticationHelper.getCurrentUser()).thenReturn(user);
+    List<String> permissionStrings = Arrays.asList(
```

Comment:
I'm missing a permission string without two '|' - by the `getApproveFacilityForecastingStats` impl it should handle them - maybe there are some other valid permission strings.

Reply:
I added other possible permission strings that may be returned by [this query.](https://github.com/OpenLMIS/openlmis-referencedata/blob/master/src/main/java/org/openlmis/referencedata/repository/RightAssignmentRepository.java#L28)